addons/planner/static/src/js/planner.js

```diff
@@ -96,6 +96,7 @@
             'click .oe_planner div[id^="planner_page"] a[href^="#planner_page"]': 'change_page',
             'click .oe_planner li a[href^="#planner_page"]': 'change_page',
             'click .oe_planner div[id^="planner_page"] button[data-pageid^="planner_page"]': 'mark_as_done',
+            'hide.bs.modal':'on_close'
```

Comment:
No it is not related to any bug. We haven't use `hide.bs.modal` event because until now we didn't need it
In planner we only having event for `planner_progress_changed` which will triggered when user make change in planner. 
But here i want to make change in Dashboard graph only then when user close the planner, not every time user make change in planner. so i just handled bootstrap's default  `hide.bs.modal` event.